src/engagement_db_to_analysis/configuration.py

```diff
@@ -35,6 +35,7 @@ class AnalysisDatasetConfiguration:
     engagement_db_datasets: [str]
     dataset_type: DatasetTypes
     raw_dataset: str
+    dataset_name: str
```

Comment:
Can you call this something like "rapid_pro_group_name" instead? (PS is it a name or an id?)

Reply:
dataset_name is more appropriate here so that we can reuse the variable in other scripts e.g automated graphs x-axis.